content/donate.adoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: "Donate to Jenkins"
+---
+
+NOTE: You can donate directly through the
```

Comment:
They're not random line breaks, they're what this text looks like when it's wrapped by default into an 80-ish column text editor